docs/2.guide/1.concepts/3.rendering.md

```diff
@@ -126,6 +126,26 @@ This will produce three files:
 
 The `200.html` and `404.html` might be useful for the hosting provider you are using.
 
+#### Skip default routes generation
+
+When prerendering a client-rendered app, Nuxt will generate `index.html`, `200.html` and `404.html` files by default. However, if you need to prevent any (or all) of these files from being generated in your build, you can use the `'prerender:generate'` hook from [Nitro](/docs/getting-started/prerendering#prerendergenerate-nitro-hook).
```

Comment:
```suggestion
When prerendering a client-rendered app, Nuxt will generate `index.html`, `200.html` and `404.html` files by default. However, if you need to prevent any (or all) of these files from being generated in your build, you can use the `'prerender:generate'` hook from [Nitro](/docs/getting-started/prerendering#prerendergenerate-nitro-hook).

```